src/AddTodoForm.jsx

```diff
@@ -0,0 +1,16 @@
+function AddTodoForm () {
+    return (
+
+    <form>
+        <label>Title</label>
+        <input id="todoTitle" htmlFor="todoTitle"></input>
```

Comment:
The `htmlFor` prop needs to be on the label instead of the input.

Reply:
done.